src/Illuminate/Container/Container.php

```diff
@@ -590,6 +590,8 @@ public function factory($abstract)
      * @param  string  $abstract
      * @param  array  $parameters
      * @return mixed
+     *
+     * @deprecated See make.
```

Comment:
Please change it to something like `@deprecated Use make() method instead`

Reply:
You could update the description that this method will be removed in 5.9